### ISSUE TITLE
Slow startup

### ISSUE THREAD

**Issue by guglovich** [flathub/com.icons8.Lunacy#9]
It takes a very long time to start up. At first it was about half a minute. Then there was no interface at all with nothing to click on for 2 minutes.

**Comment by garrett** [comment]
I agree; this is a problem. It's not an issue with the Flatpak; it's a problem with Lunacy itself. I don't think there's anything we can do that's Flatpak-specific; this Flatpak just repackages the official deb, which also exhibits the same slow startup issue.
